docs/usage/self-hosted-experimental.md

```diff
@@ -134,6 +134,7 @@ Skip initializing `RE2` for regular expressions and instead use Node-native `Reg
 ## `RENOVATE_X_MERGE_CONFIDENCE_API_BASE_URL`
 
 If set, Renovate will query this API for Merge Confidence data.
+If using Mend Renovate Enterprise and a static merge confidence token that you set via `MEND_RNV_MC_TOKEN` (or set `MEND_RNV_MC_TOKEN` to `auto`), you will need to set this variable at the server as well as the workers, if you have specified the token as a [match confidence](https://docs.renovatebot.com/configuration-options/#matchconfidence) `hostRule`, you will only need to set this variable at the workers.
```

Comment:
```suggestion

If you use Mend Renovate Enterprise and:

- have a static merge confidence token that you set via `MEND_RNV_MC_TOKEN`
- _or_ set `MEND_RNV_MC_TOKEN` to `auto`

Then you must set this variable at the _server_ and the _workers_.

But if you have specified the token as a [`matchConfidence`](https://docs.renovatebot.com/configuration-options/#matchconfidence) `hostRule`, you only need to set this variable at the _workers_.
```

Changes:

- use a list to split complex clauses
- _emphasize_ the key differences
- apply code styling for the `matchConfidence` config option name
- use the word must to indicate requirements

I'll let a maintainer check the content for technical accuracy.

The link to the doc should be changed to a relative link, I think.